docs/userGuide/deployingTheSite.md

```diff
@@ -137,8 +145,6 @@ Since May 2018, Travis CI has been [undergoing migration to `travis-ci.com`](htt
 
 ##### Configuring your repository in Travis CI
 
-1. [Generate a GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token) with **repo** permissions. Take note of the generated token - you will not be able to see it again once you navigate away from the page.
-<include src="screenshot.md" boilerplate var-alt="GitHub Token Repo Permissions" var-file="githubTokenRepoPermissions.png" inline />
 1. [Add an environment variable in Travis CI](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings) named `GITHUB_TOKEN`, with the value set to the personal access token generated in the previous step. ==Ensure that _Display value in the build log_ is set to _Off_.==
```

Comment:
Regarding the "previous step", I think we should link it back to the *personal access token section* above.

Reply:
Thanks for the catch @wxwxwxwx9 !